Command/BaseBootstrapSymlinkCommand.php

```diff
@@ -62,7 +62,11 @@ public static function checkSymlink($symlinkTarget, $symlinkName, $forceSymlink
                 if (!$forceSymlink) {
                     throw new \Exception(sprintf('Symlink "%s" points to "%s" instead of "%s"', $symlinkName, $linkTarget, $symlinkTarget));
                 }
-                unlink($symlinkName);
+                if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' && is_dir($linkTarget)) {
```

Comment:
Strange thing - took the check from:
https://github.com/symfony/symfony-standard/blob/2.8/app/SymfonyRequirements.php#L703